src/utilities/utilities.js

```diff
@@ -76,34 +92,27 @@ export const toastInfoOptions = {
   }
 }
 
-const PrysmScanLink = (token) => {
-  return (`${PRYSM_APP_URL}/profile/${token.owner.id}/achievements`);
-};
-
-const PoapExploreLink = (token) => {
-  return (`${POAP_EXPLORE_APP_URL}/${token.owner.id}`);
-};
-
-const RainbowLink = (token) => {
-  return (`${RAINBOW_APP_URL}/${token.owner.id}`);
-};
-
-const MazuryLink = (token) => {
-  return (`${MAZURY_APP_URL}/people/${token.owner.id}`);
-};
-
-const WithTallyLink = (token) => {
-  return (`${WITHTALLY_APP_URL}/voter/${token.owner.id}`)
+export const externalLinkSetter = (owner_id, name) => {
+  const collectionLinks = {
+    'PRYSM': `${PRYSM_APP_URL}/profile/${owner_id}/achievements`,
```

Comment:
the keys have must be a constant, and they have to be used in the collectionlLinks array as well.